package.json

```diff
@@ -1,18 +1,23 @@
 {
   "name": "model-viewer",
   "private": true,
-  "version": "0.0.0",
+  "version": "0.1.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
-    "build": "tsc && vite build",
+    "tauri:dev": "GDK_BACKEND=x11 pnpm tauri dev",
```

Comment:
Added `GDK_BACKEND=x11` env variable to get rid of annoying warning message on Windows.